requirements.txt

```diff
@@ -2,4 +2,5 @@ fire==0.4.0
 requests==2.25.1
 xmltodict==0.12.0
 pylint==2.6.0
-bandit==1.7.0
\ No newline at end of file
+bandit==1.7.0
+defusedxml==0.7.1
```

Comment:
qu'est-ce que c'est que ça?

Reply:
https://docs.python.org/3.6/library/xml.html#the-defusedxml-and-defusedexpat-packages

okay